modules/rosa-sts-creating-cluster.adoc

```diff
@@ -154,7 +163,15 @@ $ oc adm release extract quay.io/openshift-release-dev/ocp-release:${version:0:3
 $ cat credrequests/0000*.yaml > credrequests/${version:0:3}.yaml
 $ rm -f credrequests/0000*.yaml
 ----
-.. Create the IAM roles.
++
+[NOTE]
+====
+This requires the OpenShift CLI (`oc`), version 4.7.9 or greater. You can
```

Comment:
Right now, cross-references are not supported from modules. So you may want to just give them the download command instead.